.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+test.md
+package-lock.json
+package.json
```

Comment:
- [ ] [OPTIONAL] These files are not supposed to be ignored kindly ensure you remove them from .gitignore.

Reply:
Thank you for the feedback